config/sprites/topographic/building_pnt_camp_1.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><defs><style>.cls-1{fill:#fff;stroke:#231f20;stroke-miterlimit:10;stroke-width:0.8px;}.cls-2{fill:#231f20;}</style></defs><circle class="cls-1" cx="8.07" cy="8" r="7.41"/><polygon class="cls-2" points="1.93 10.58 5.06 5.42 10.47 5.42 14.22 10.58 9.9 10.58 9.9 5.89 7.25 10.58 1.93 10.58"/></svg>
```

Comment:
We have camp 1 and camp 2, two icons for camp?

Reply:
yeah just 2 drafts, will decide which one we like more and keep that one